components/transfer/TransferConfirmModal.vue

```diff
@@ -265,3 +272,18 @@ const isExpandList = ref(false)
   }
 }
 </style>
+<style lang="scss">
+/* trick to implement 'parent-selector' */
+div:has(> .parent-z-index-auto) {
+  z-index: auto !important;
+}
+
+div:has(> .parent-z-index-auto.mobile) {
+  .o-tip__content {
+    transform: translateX(calc(-100vw + 2rem));
+  }
+  .o-tip__arrow {
+    transform: translateX(calc(50vw - 2rem));
+  }
+}
```

Comment:
Can you remove this logic into NeoTooltip component? I think it would be reused in other places in the future.